<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcd1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>로또생성하기</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
